explore/src/main/scala/explore/SideTabs.scala

```diff
@@ -32,53 +34,65 @@ object SideTabs {
       .builder[Props]
       .stateless
       .render_P { p =>
-        val tabsL = p.tabs.get.toNel
-        val focus = p.tabs.get.focus
+        AppCtx.withCtx { ctx =>
+          val tabsL = p.tabs.get.toNel
+          val focus = p.tabs.get.focus
 
-        def tabButton(tab: AppTab): Button =
-          Button(active = tab === focus,
-                 clazz = ExploreStyles.SideButton,
-                 onClick = p.tabs.mod(z => z.findFocus(_ === tab).getOrElse(z)).runAsyncCB
-          )(tab.title)
+          def tabButton(tab:          AppTab): Button       =
+            Button(
+              as = <.a,
+              active = tab === focus,
+              clazz = ExploreStyles.SideButton,
+              onClickE = (e: ReactMouseEvent, _: ButtonProps) =>
+                e.preventDefaultCB *> p.tabs
+                  .mod(z => z.findFocus(_ === tab).getOrElse(z))
+                  .runAsyncCB
```

Comment:
I would have expected the `href` to be enough so that we could remove the `onClick`. It doesn't trigger the router?

Reply:
If it's a link, does it even have to be a Button? Well, other than the fact that we are using ButtonGroup to group them. Ignore this if the onClickE is necessary.

Reply:
Without the onClick it will go to that route but with a page reload. 
With respect to making it a plain link, it is possible but it would require extra css to restore the current look

Reply:
Oh, interesting. So, this combination allows opening in another tab, etc., but prevents a page reload when clicking. Good job figuring that out - it seems non-obvious to me.

Reply:
Isn't e.preventDefaultCB enough as in TargetObsList?

Reply:
Thanks for spotting this, `preventDefaultCB` isn't enough

Reply:
Fixed, and I realized I'm blocking the regular cmd+click shortcuts, so now it is handled as well.
We may want to refactor this into a shared method for handling these